### PR TITLE
Message chats

### PR DESCRIPTION
**Pull request by RubenMattan**
Merge jobs branch first

### CODE REVIEW

File: rogerthat/src/main/java/com/mobicage/rogerthat/osa/demo2/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+       specific language governing permissions and limitations
+       under the License.
+ */
+
+package com.mobicage.rogerthat.osa.demo2;
+
+import android.os.Bundle;
+import org.apache.cordova.*;
+
+public class MainActivity extends CordovaActivity
```
Comment (lucasvanhalst) [inline rogerthat/src/main/java/com/mobicage/rogerthat/osa/demo2/MainActivity.java]: Not sure what this file is for?

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/messaging/ChatInfoMembersActivity.java [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+                } else if ("make_group_admin".equals(type)) {
+                    mCallbackKey = UUID.randomUUID().toString();
+                    showLoadingDialog();
+                    mMadeChanges = true;
+                    mMessagingPlugin.makeMembersAdminOfConversation(mCallbackKey, mChatKey, new String[] { item.email });
+                } else if ("remove_member".equals(type)) {
+                    mCallbackKey = UUID.randomUUID().toString();
+                    showLoadingDialog();
+                    mMadeChanges = true;
+                    mMessagingPlugin.removeMembersFromConversation(mCallbackKey, mChatKey, new String[] { item.email });
```
Comment (lucasvanhalst) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/messaging/ChatInfoMembersActivity.java]: refactor so we don't have the same code 3 times

File: rogerthat/src/main/java/com/mobicage/rogerthat/plugins/messaging/ChatInfoActivity.java [hunk 3]
```diff
@@ -137,13 +441,247 @@ public View getView(int position, View convertView, ViewGroup parent) {
...
+            tag.put("type", "item");
+            tag.put("item", item);
+            v.setTag(tag);
+
+            final ImageView avatarView = v.findViewById(R.id.member_avatar);
+            avatarView.setImageBitmap(null);
+            ViewGroup.LayoutParams lp = avatarView.getLayoutParams();
+            lp.width = lp.height = UIUtils.convertDipToPixels(ChatInfoActivity.this, 50);
+
+            Picasso.get().load(item.avatar_url).resize(50, 50).into(new com.squareup.picasso.Target() {
```
Comment (lucasvanhalst) [inline rogerthat/src/main/java/com/mobicage/rogerthat/plugins/messaging/ChatInfoActivity.java]: I'd use this .transform instead, more reusable and readable https://stackoverflow.com/a/30707381/2396593
 (same for the other file which has this copy pasted)

**Comment by RubenMattan** [comment]
> * Remove member -> send some messages -> add same member again -> messages send in the time you weren't in the group aren't visible to said member. Not sure if intended.
> * Remove member -> Rename group -> add same member -> member still sees old group name / avatar

Created issues on backend repo -> https://github.com/our-city-app/mobicage-backend/issues/130

**Pull request merged**